src/site/_includes/components/Widget.js

```diff
@@ -0,0 +1,25 @@
+const patterns = require('../../_data/patterns').patterns;
+
+/**
+ * @fileoverview A component to display an interactive demo,
+ * specified as part of a code pattern in /patterns/ directory,
+ * as an iframe embed.
+ */
+
+/**
+ * @param {string} patternId Id of the Code Pattern to be displayed.
+ */
+module.exports = (patternId, height) => {
+  const pattern = patterns[patternId];
+  if (!pattern) {
+    return '';
+  }
+  return `<div class="widget" style="min-height: ${height}px">
+      <iframe
+        src="${pattern.demo}"
+        title="Demo"
+        height="${height}"
+        width="100%"
+        scrolling="no"></iframe>
```

Comment:
What do you think about adding https://web.dev/iframe-lazy-loading/ to this, either by default or via a `loading` parameter, similar to https://github.com/GoogleChrome/webdev-infra/blob/main/shortcodes/Img.js#L113-L115?

Reply:
I made it lazy by default, if we get use cases where this is not enough I'll make it a param.